app/javascript/styles/mastodon/components.scss

```diff
@@ -10939,3 +10941,40 @@ noscript {
 .lists-scrollable {
   min-height: 50vh;
 }
+
+.featured-carousel {
+  background: var(--surface-background-color);
+  overflow: hidden;
+  border-bottom: 1px solid var(--background-border-color);
+
+  &__slides {
+    display: flex;
+    flex-wrap: nowrap;
+    touch-action: pan-x;
+  }
+
+  &__slide {
+    flex: 0 0 auto;
+    flex-basis: 100%;
+  }
+
+  .status {
+    border-bottom: 0;
+  }
```

Comment:
I needed a similar override recently for Quote posts. Maybe the Status component should just have a `withBottomBorder = true` prop.